CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## Next
+
+### ActionMenu
+
+- Action menu now uses button content.
```

Comment:
"button content" is internal, change to "now supports same props as button" or similar.